.github/workflows/CreatePullRequest.yml

```diff
@@ -0,0 +1,21 @@
+name: Create Pull Request
+
+on:
+  pull_request:
+    types: ["opened"]
+
+jobs:
+  CreateCardForStandalonePR_job:
+    name: Assign PR to the author and create a Kanban card
+    runs-on: ubuntu-latest
+    # Single quotes must be used here https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#literals
+    # PRs from forks don't have required token authorization
+    # Dependabot works directly under our repository, but doesn't have enough priviledges to create project card
+    if: |
+        github.event.pull_request.head.repo.full_name == github.repository
+        && github.event.sender.type != 'Bot'
+    steps:
+      - uses: sonarsource/gh-action-lt-backlog/CreateCardForStandalonePR@v1
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          column-id: 6183856
```

Comment:
Please extract a constant to have a name for the column id.

Reply:
Same as on sonar-dotnet